python/nnvm/top/tensor.py

```diff
@@ -37,11 +36,42 @@ def _compute(attrs, x, _):
         return f(x[0], x[1])
     return _compute
 
-
 _fschedule_injective = tvm.convert(_schedule_injective)
 _fschedule_broadcast = _fschedule_injective
 _fschedule_elemwise = _fschedule_injective
 
+# lshift
+@reg.register_compute("left_shift")
```

Comment:
this is not consistent with numpy left_shift. To make it consistent with current add, sub, we should name it ```__lshift_scalar__```